### ISSUE TITLE
Prepare data for bar charts for ASNs on country page

### ISSUE THREAD

**Issue by zelima** [cybergreen-net/stats-new#150]
Prepare data for ASN bar charts 
- Get fist 10 asn with most count for given country and risk
- List of graphs to be rendered (risks)
- List of URLs to API endpoints for graphs (Total, given counrty)
- Type of graph (line, bar)

**Comment by zelima** [comment]
This needs some more planing fist (probably on react app side as well), before starting implementation. 
Moving to next milestone

**Comment by rufuspollock** [comment]
INVALID / WONTFIX. Obsoleted now in favour of new epic #161
